### DISCUSSION TITLE
How can i delete db file after connection.close()

### DISCUSSION THREAD

**Discussion by binggo8322** [xerial/sqlite-jdbc#713] [Q&A]
I am in Java (kotlin).  Version: org.xerial:sqlite-jdbc:3.36.0.2
I would like to delete the file db after each test.
When I try to delete the file I get an error. "The process cannot access the file because it is being used by another process"

Code:
var connection : Connection = (dataSource as SQLiteDataSource).connection
((connection) as SQLiteConnection).close()

**Comment by gotson** [comment]
You should use in memory database for tests.
